boards/arm/stm32g0316_disco/pinmux.c

```diff
@@ -9,8 +9,6 @@
 #include <stm32_ll_bus.h>
 #include <stm32_ll_system.h>
 
-#include <pinmux/stm32/pinmux_stm32.h>
```

Comment:
To be replaced by `#include <pinmux/pinmux_stm32.h>` ?
Or no more necessary ?

Reply:
From what I see, it doesn't need this header anymore.